examples/index.html

```diff
@@ -302,6 +302,9 @@ <h4><b>By entering co-ordinates :</b></h4>
         </div>
         <!-- end multi module -->
       </div>
+      <div class="bug-btn-wrapper">
+        <a href="https://publiclab.org/post?n=23853&title=Give%20this%20bug%20a%20title%20here" class="bug-btn" target="_blank">Found an issue with the editor? Report a bug.</a>
```

Comment:
Here, could we make the title say either nothing (user can input their own) or "Editor bug report: " and they can add their own after that?